examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -743,35 +743,11 @@ function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
         )
         @. Yₜ.c.ρe_tot +=
             ᶜdivᵥ(ᶠK_E * ᶠinterp(ᶜρ) * ᶠgradᵥ((Y.c.ρe_tot + ᶜp) / ᶜρ))
-    elseif :ρe_int in propertynames(Y.c)
```

Comment:
@charleskawczynski Maybe instead of removing it, we can throw an error here and stop the simulation if we are using `ρe_int`. Would this be better? If so how should I do this?

Reply:
We can try throwing an error. Maybe add in one place and make sure that JET doesn't complain before going and adding it everywhere. Alternatively, we could just add the assertion/error when creating `Y` (that's actually probably much simpler)

Reply:
Maybe we can do it in `get_model_spec` [here](https://github.com/CliMA/ClimaAtmos.jl/blob/227c9039ff341a081492c7d4a28f2ea8a5cf9e0f/examples/hybrid/types.jl#L102)? Or do you think it's better to do it [here](https://github.com/CliMA/ClimaAtmos.jl/blob/227c9039ff341a081492c7d4a28f2ea8a5cf9e0f/examples/hybrid/sphere/baroclinic_wave_utilities.jl#L27) when creating Y?

Reply:
Ah, true, we could error in model spec. Either seems fine